example/pubspec.yaml

```diff
@@ -26,7 +26,7 @@ dependencies:
 
   reactive_forms:
     path: ../
-
+  intl: ^0.16.1
```

Comment:
Do you think is a good idea to add a dependency to another library in Reactive Forms? I haven't implement this Date Value Accessor using DateFormat for that reason, because I'm concerned that users of Reactive Forms become to having some compatibility issues of the **intl** with other libraries. What do you think about it?